### ISSUE TITLE
After introducing quicklist (recent Redis unstable), Jedis unit test is failing

### ISSUE THREAD

**Issue by HeartSaVioR** [xetorthio/jedis#845]
- What change makes unit test failing?

quicklist, proposed from https://github.com/antirez/redis/pull/2143, merged by https://github.com/antirez/redis/commit/05ba119fbb9b0e05b7cca83bee8b01041e8877f2

- Build output (short ver.)

```
Results :

Failed tests:
  ObjectCommandsTest.objectEncoding:28->Assert.assertEquals:144->Assert.assertEquals:115 expected:<[zip]list> but was:<[quick]list>
```

Seems like we can just change expected value and everything would be okay.

**Comment by HeartSaVioR** [comment]
We can also remove that test, or just compare it with *list.
@xetorthio @marcosnils Which one is proper way for you?

**Comment by marcosnils** [comment]
@HeartSaVioR as Jedis tests should always be run against unstable Redis branch I guess we can change the assertion value to `quicklist` and we should be ok.

**Comment by nykolaslima** [comment]
@marcosnils already fixed it in #846. Can we close this?

**Comment by HeartSaVioR** [comment]
@marcosnils Thanks for following up!
